### PR TITLE
update: refactored notation list command

### PR DESCRIPTION
**Pull request by patrickzheng200**
This PR refactors the `notation list` command reducing the number of calls to getSignatureRepository() and repo.Resolve().

### CODE REVIEW

File: cmd/notation/manifest.go [hunk 1]
```diff
@@ -21,14 +25,12 @@ func getManifestDescriptor(ctx context.Context, opts *SecureFlagOpts, reference
...
-		return ocispec.Descriptor{}, registry.Reference{}, err
-	}
 
-	manifestDesc, err := repo.Resolve(ctx, ref.Reference)
+	manifestDesc, err := sigRepo.Resolve(ctx, ref.Reference)
 	if err != nil {
 		return ocispec.Descriptor{}, registry.Reference{}, err
 	}
+
+	logger.Infof("Reference resolved to manifest descriptor: {MediaType:%v, Digest:%v, Size:%v}", manifestDesc.MediaType, manifestDesc.Digest, manifestDesc.Size)
```
Comment (priteshbandi) [inline cmd/notation/manifest.go:34]:
```suggestion
	logger.Infof("Reference %s resolved to manifest descriptor: %+v", ref.Reference, manifestDesc)
```

**Review by priteshbandi — APPROVED** [review]
LGTM with one suggestion

File: cmd/notation/manifest.go (hunk 1 above)
Comment (patrickzheng200) [reply cmd/notation/manifest.go]: Updated.

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**